src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -970,6 +973,11 @@ def customizePerJob(self, job):
         if job.get('inputDatasetLocations'):
             jdl.append('+DESIRED_CMSDataLocations = "%s"\n' % ','.join(job['inputDatasetLocations']))
 
+        # HighIO jobs
+        highIOjob = job.get('highIOjob')
+        if highIOjob:
+            jdl.append('+Requestioslots = 1\n')
```

Comment:
So it is trying to match high io slot first among the possible site then if there no site with high io slots it will still match the other possible sites, correct?